docker-compose.yml

```diff
@@ -22,5 +22,17 @@ services:
       - postgres
     restart: unless-stopped
 
+  op-succinct-celestia:
+    build:
+      context: .
+      dockerfile: ./validity/Dockerfile.celestia
+    env_file:
+      - ${ENV_FILE:-.env.celestia}
+    environment:
+      - DATABASE_URL=postgres://op-succinct@postgres:5432/op-succinct
+    depends_on:
+      - postgres
+    restart: unless-stopped
+
```

Comment:
Adding this to the Docker compose doesn't seem correct. You should have a separate `op-succinct-celestia` docker compose file.